code/modules/antagonists/traitor/equipment/contractor.dm

```diff
@@ -170,6 +255,8 @@
 
 	hub.purchased_items.Add(src)
 
+	user.playsound_local(user, 'sound/machines/uplinkpurchase.ogg', 100)
```

Comment:
have you checked this in game? 100 might be _very_ loud depending on the sound

Reply:
Yeah, the sound's fine, not sure if it's an issue with me uploading quieter sounds or something but 100 is needed.

Reply:
sounds should be probably be normalized at some point but it's not really a major concern of this PR.